src/App.tsx

```diff
@@ -16,9 +20,7 @@ import Markdown from "./components/Markdown";
 
 function MapHack({ setMap }: { setMap: any }): null {
     const map = useMap();
-    useEffect(() => {
-        setMap(map);
-    });
+    setMap(map);
```

Comment:
Das `useEffect` brauch es schon, sonst kommt es zu folgendem Fehler in der Konsole:
```
Warning: Cannot update a component (`App`) while rendering a different component (`MapHack`). To locate the bad setState() call inside `MapHack`, follow the stack trace as described in https://reactjs.org/link/setstate-in-render
MapHack@http://localhost:3000/static/js/bundle.js:60:17
div
MapContainerComponent@http://localhost:3000/static/js/bundle.js:117756:31
div
App@http://localhost:3000/static/js/bundle.js:74:76```